programs/mango-v4/src/lib.rs

```diff
@@ -545,9 +548,10 @@ pub mod mango_v4 {
             stable_price_delay_growth_limit_opt,
             stable_price_growth_limit_opt,
             settle_pnl_limit_factor_opt,
-            settle_pnl_limit_window_size_ts,
+            settle_pnl_limit_window_size_ts_opt,
```

Comment:
I think this is missing the corresponding ts client change

Reply:
The ts client passes:
```
        params.settlePnlLimitWindowSize !== null
          ? new BN(params.settlePnlLimitWindowSize)
          : null,
```
so it looks like it doesn't use the _opt name anyway. Did you mean something else?